protoutil/blockutils.go

```diff
@@ -242,18 +261,31 @@ type policy interface { // copied from common.policies to avoid circular import.
 	EvaluateSignedData(signatureSet []*SignedData) error
 }
 
+
+// BlockSignatureVerifier returns a BlockVerifierFunc that validates the signatures of a block based on the provided policy and consenter set.
+// It checks the metadata of the block to retrieve and verify the signatures against the given policy.
```

Comment:
For functions that are missing a function comment, I agree they should be added. But I would keep them to one or two sentences (like above), and avoid the full listing of parameters (like below).